token/dig/sdk.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sdk
+
+import (
+	"context"
+	errors2 "errors"
+	"time"
+
+	digutils "github.com/hyperledger-labs/fabric-smart-client/platform/common/utils/dig"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric/core"
+	fabricsdk "github.com/hyperledger-labs/fabric-smart-client/platform/fabric/dig"
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/driver"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/flogging"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/kvs"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/tracing"
+	"github.com/hyperledger-labs/fabric-token-sdk/token"
+	core2 "github.com/hyperledger-labs/fabric-token-sdk/token/core"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/core/common/logging"
+	_ "github.com/hyperledger-labs/fabric-token-sdk/token/core/fabtoken/driver"
```

Comment:
the other sdk.go doesn't have any blind imports. Why do we need them here?

Reply:
Oh no we don't need them. This version was based on a previous version of the sdk that had the blind imports. Will remove them.